### PR TITLE
Rebranding [in-progress]

### PR DESCRIPTION
**Pull request by mDuo13**
Addresses **DOC-994** (rebranding) and **DOC-519** (migrate consensus articles)

### CODE REVIEW

File: content/reference-rippled.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # rippled #
 
-The core peer-to-peer server that manages the Ripple Consensus Ledger (RCL) is called `rippled`. Each `rippled` server connects to a network of peers, relays cryptographically signed transactions, and maintains a local copy of the complete shared global ledger. The source code for `rippled` is written in C++, and is [available on GitHub under an open-source license](https://github.com/ripple/rippled).
+The core peer-to-peer server that manages the XRP Ledger (XRP Ledger) is called `rippled`. Each `rippled` server connects to a network of peers, relays cryptographically signed transactions, and maintains a local copy of the complete shared global ledger. The source code for `rippled` is written in C++, and is [available on GitHub under an open-source license](https://github.com/ripple/rippled).
```
Comment (ryangyoung) [inline content/reference-rippled.md]: > XRP Ledger (XRP Ledger)

We probably don't need the parenthetical anymore.

File: content/reference-rippled.md [hunk 2]
```diff
@@ -499,7 +499,7 @@ The format of the `marker` field is intentionally undefined. Each server can def
 
 ## Modifying the Ledger ##
 
-All changes to the Ripple Consensus Ledger (RCL) happen as the result of transactions. The only API methods that can change the contents of the RCL are the [`submit` command](#submit) and the [`submit_multisigned` command](#submit-multisigned). Most other methods represent different ways to view the data represented in the RCL. The remaining commands generate data for your convenience. (The [`wallet_propose`](#wallet-propose), [`path_find`](#path-find), and [`random`](#random) commands fall into this category.)
+All changes to the XRP Ledger (XRP Ledger) happen as the result of transactions. The only API methods that can change the contents of the RCL are the [`submit` command](#submit) and the [`submit_multisigned` command](#submit-multisigned). Most other methods represent different ways to view the data represented in the RCL. The remaining commands generate data for your convenience. (The [`wallet_propose`](#wallet-propose), [`path_find`](#path-find), and [`random`](#random) commands fall into this category.)
```
Comment (ryangyoung) [inline content/reference-rippled.md]: A few "RCL" instances in this paragraph.

Comment (ryangyoung) [reply content/reference-rippled.md]: Also, another "XRP Ledger (XRP Ledger)" instance.

File: content/reference-transaction-format.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ A _Transaction_ is the only way to modify the Ripple Ledger. Transactions are on
```
Comment (ryangyoung) [inline content/reference-transaction-format.md:11]: We should change "Ripple Ledger" in [line 3](https://github.com/ripple/ripple-dev-portal/pull/263/files#diff-d99a4557e170059e331d3ce7b5ee7338R3) to "XRP Ledger".

File: content/reference-transaction-format.md [hunk 4]
```diff
@@ -1079,9 +1079,9 @@ Create or modify a trust line linking two accounts.
 
 ### Trust Limits ###
 
-All balances on the Ripple Consensus Ledger (RCL), except for XRP, represent value owed in the world outside the Ripple Ledger. The address that issues those funds in Ripple (identified by the `issuer` field of the `LimitAmount` object) is expected to pay the balance back, outside of the Ripple Consensus Ledger, when users redeem their Ripple balances by returning them to the issuer.
+All balances on the XRP Ledger (XRP Ledger), except for XRP, represent value owed in the world outside the Ripple Ledger. The address that issues those funds in Ripple (identified by the `issuer` field of the `LimitAmount` object) is expected to pay the balance back, outside of the Ripple Consensus Ledger, when users redeem their Ripple balances by returning them to the issuer.
```
Comment (ryangyoung) [inline content/reference-transaction-format.md]: >  XRP Ledger (XRP Ledger)

Drop the parens.

Comment (ryangyoung) [reply content/reference-transaction-format.md]: Also, should change "Ripple Consensus Ledger" and "Ripple Ledger" here.

File: content/reference-transaction-format.md [hunk 5]
```diff
@@ -1079,9 +1079,9 @@ Create or modify a trust line linking two accounts.
...
+Since a computer program cannot force a someone to keep a promise and not default in real life, trust lines represent a way of configuring how much you trust an issuer to hold on your behalf. Since a large, reputable financial institution is more likely to be able to pay you back than, say, your broke roommate, you can set different limits on each trust line, to indicate the maximum amount you are willing to let the issuer "owe" you in the XRP Ledger. If the issuer defaults or goes out of business, you can lose up to that much money because the balances you hold in the XRP Ledger can no longer be exchanged for equivalent balances elsewhere. (You can still keep or trade the issuances in the RCL, but they no longer have any reason to be worth anything.)
```
Comment (ryangyoung) [inline content/reference-transaction-format.md]: > You can still keep or trade the issuances in the RCL

"RCL" ^^

File: content/snippets/gateways-intro.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
 
-* An **Issuing Gateway** receives money (or other assets of value) outside of Ripple, and creates issuances in the Ripple Consensus Ledger. This provides a direct way for customers to get money in and out of the RCL. All currencies in the Ripple Consensus Ledger, except for XRP, take the form of issuances tied to a specific issuing gateway.
-* A **Private Exchange** holds XRP and lets its customers buy and sell that XRP in its own system. Most cryptocurrencies rely on private exchanges to provide a market for the cryptocurrency, but the Ripple Consensus Ledger has a currency exchange built into the protocol itself.
-* **Merchants** accept payment within the Ripple Consensus Ledger in exchange for goods and services in the outside world. Currently, Ripple does not have widespread support for merchant operations.
+* An **Issuing Gateway** receives money (or other assets of value) outside of Ripple, and creates issuances in the XRP Ledger. This provides a direct way for customers to get money in and out of the XRP Ledger. All currencies in the Ripple Consensus Ledger, except for XRP, take the form of issuances tied to a specific issuing gateway.
```
Comment (ryangyoung) [inline content/snippets/gateways-intro.md]: > Ripple Consensus Ledger

File: content/reference-ledger-format.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
+The XRP Ledger is a shared, global ledger that is open to all. Individual participants can trust the integrity of the ledger without having to trust any single institution to manage it. The `rippled` server software accomplishes this by managing a ledger database that can only be updated according to very specific rules. Each instance of `rippled` keeps a full copy of the ledger, and the peer-to-peer network of `rippled` servers distributes candidate transactions among themselves. The consensus process determines which transactions get applied to each new version of the ledger. See also: [The Consensus Process](https://ripple.com/build/ripple-ledger-consensus-process/).
 
 ![Diagram: Each ledger is the result of applying transactions to the previous ledger version.](img/ledger-process.png)
```
Comment (ryangyoung) [inline content/reference-ledger-format.md:6]: On [line 56](https://github.com/ripple/ripple-dev-portal/pull/263/files#diff-c2794d8e179e72859b7c8da43272467cR56), there's a "consensus ledger" instance we should change.

File: content/data_types/public_key.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-The Ripple Consensus Ledger (RCL) uses public keys to verify cryptographic signatures in several places:
+The XRP Ledger (XRP Ledger) uses public keys to verify cryptographic signatures in several places:
```
Comment (ryangyoung) [inline content/data_types/public_key.md]: > XRP Ledger (XRP Ledger)

Parens again

File: content/tutorial-paychan.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ Payment Channels are an advanced feature for sending "asynchronous" XRP payments
 
 ## Background
 
-The process of using a payment channel always involves two parties, a payer and a payee. The payer is an individual person or institution using the Ripple Consensus Ledger who is a customer of the payee. The payee is a person or business who is doing business on the Ripple Consensus Ledger (RCL), receiving XRP as payment for goods or services.
+The process of using a payment channel always involves two parties, a payer and a payee. The payer is an individual person or institution using the XRP Ledger who is a customer of the payee. The payee is a person or business who is doing business on the Ripple Consensus Ledger (XRP Ledger), receiving XRP as payment for goods or services.
```
Comment (ryangyoung) [inline content/tutorial-paychan.md]: > Ripple Consensus Ledger (XRP Ledger)

mismatch here ^

File: content/reference-ledger-format.md (hunk 7 above)
Comment (mDuo13) [reply content/reference-ledger-format.md:6]: The line-56 "consensus ledger" (lowercase-C) is correct as-is, but I've changed it to "consensus ledger version" to be a little clearer that we're talking about "the version of the ledger built as a result of the consensus process"

**Pull request merged**